meetings/2024-06-12.md

```diff
@@ -28,11 +35,123 @@
 
 
 ### Scribes
-* [name]
+* Hadrian Zbarcea
+* elf Pavlik
 
 ### Introductions
 
+* Erich, Stony Brook University  in NYC. Biometric informatics. Developing a pathology management system. Interested for that development.
+
 ## Topics
 
 ### WG status
 
+* PAC: not much to report, some questions from the TAG, will get reviewed soon enough. There was no security review on the previous charter. We are asking for a security review.
+* HZ: anything we can do at this point.
+* PAC: the ball is on their side. Make sure the ball keeps rolling.
+* PAC: Hard to say how much longer it will take. Still optimistic. Wouldn't risk giving an estimate. Hope by the end of June.
+
+### TPAC Meeting
+
+* VB: Our selected slot was not available so the meeting was moved to [Thursday September 26th at 18:00 UTC](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=9&day=26&hour=18&min=0&sec=0&p1=783&p2=37) 
+* eP: can we add it to the Solid Calendar
+* PAC: I think it'll be added automatically
+
+ACTION: HZ to follow up and make sure it gets done
+
+* PAC: I have an issue, as I already have a number of conflicts. If other people wish to move it, I'd be in favor of that; Friday would be better for me. This is the 2nd slot in the morning, maybe move it even earlier.
+* eP: Can we do a doodle to find a spot? Share the result with organizer to get maybe a second option.
+
+ACTION: PAC to create a doodle.
+  → https://evento.renater.fr/survey/solid-cg-meeting-at-tpac-2025-oat4znsh
+
+* PAC: do we put all slots, or filter first?
+* eP: prefer to add all and filter later
+* PAC: maybe give a deadline.
+* VB: (following minutes but cannot join): Our options are limited to what's available. Event will be added automatically to our calendar. Our first option of Monday was not available. Tuesday was not good for some people. Feel free to add all slots and we can ask what's possible but deadline was June 10th to submit, so our choices might no longer be available at this point. Let me know and I'll check. 
+* eP: sharing with the group what VB is writing.
+* PAC: we're past the deadline anyway
+* eP: 48 hour should suffice. we cand send list mail and @room mention on matrix (any chair can do it)
+* HZ: agreed 
+* VB: Perhaps focus on Thu and Fri, since we already know Mon and Tue don't work. 
+
+### initial WebID Document considerations
+
+URL: https://github.com/solid/security-considerations/pull/9
+
+* HZ: Jacopo is no longer a chair of WebID CG, not sure how it impacts Solid
+* eP: STM a week ago, JZ, VB, MDJ present. What's the criteria of accepting a proposal? For me, is to not conflict with the Solid protocol. Nobody showed me any problem with conformance. Discussing counter measures for attacks. My impression is that sometimes we put too much in a PR. If someone things something more needs to be added, they can create a new PR. IMHO this PR is ready to merge. If later we decide we can have more normative requirements, it can be changed.
```

Comment:
```suggestion
* eP: Solid Team Meeting a week ago; JZ, VB, MDJ present. What are the criteria for accepting a proposal? For me, it should not conflict with the Solid Protocol. Nobody showed me any problem with conformance. Discussing countermeasures for attacks. My impression is that sometimes we put too much in one PR. If someone thinks something more needs to be added, they can create a new PR. IMHO, this PR is ready to merge. If later we decide we want/need more normative requirements, it can be changed.
```